src/server.c

```diff
@@ -5649,6 +5650,7 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "client_recent_max_output_buffer:%zu\r\n", maxout,
             "blocked_clients:%d\r\n", server.blocked_clients,
             "tracking_clients:%d\r\n", server.tracking_clients,
+            "pubsub_clients:%d\r\n", server.pubsub_clients,
```

Comment:
should this be `subscribe_clients`?
i.e. we're only counting subscribers.
on the other hand, this framework in redis is called "pubsub"

Reply:
I aligned this with the client flag, we are counting clients in "pubsub mode".

Reply:
Naming is a challenging problem...

I prefer `pubsub_clients`, We refer to it as "pubsub" like in the `client-output-buffer-limit` configuration and client commands. Moreover, we also have "psubscribe" and "ssubscribe" commands. The term "subscribe_clients" may cause confusion as well.